routing/bandwidth.go

```diff
@@ -18,7 +20,20 @@ type bandwidthHints interface {
 	// will be used. If the channel is unavailable, a zero amount is
 	// returned.
 	availableChanBandwidth(channelID uint64,
-		amount lnwire.MilliSatoshi) (lnwire.MilliSatoshi, bool)
+		amount lnwire.MilliSatoshi,
+		blobs ...tlv.Blob) (lnwire.MilliSatoshi, bool)
```

Comment:
Is this the HTLC blob? Or all the channel blobs? Could be a `fn.Option[tlv.Blob]` if it's the HTLC blobs.